solutions/pandas_tests/Makefile

```diff
@@ -0,0 +1,57 @@
+# Some tests in files listed in tests.partially_passed fail/ error out on execution.
+# These failures and error are caused by one of below:
+# 1. pypaper linux specific errors, documented
+# here - https://pyperclip.readthedocs.io/en/latest/#not-implemented-error
+# 2. s3fs error (which is currently an optional file handling mechanism in pandas)
+# https://bleepcoder.com/evalml/557744402/unit-test-fails-after-upgrading-to-pandas-1-0-0
+#
+# NOTE: Running the test as collection of individual test files creates ModuleNotFoundError
+# for certain test files eg: pandas/tests/extension/arrow/arrays.py.
+
+TOP = $(abspath ../..)
+include $(TOP)/defs.mak
+
+APPBUILDER=$(TOP)/scripts/appbuilder
+PYTHON3 = /usr/local/bin/python3
+PYTEST_CMD = $(PYTHON3) -m pytest
+PYTEST_INI_FILE = pytest.ini
+PYTEST_OPTS = -s -q --no-header --skip-slow --skip-db --skip-network
+
+OPTS = --max-affinity-cpus=4 --app-config-path config.json
+
+ifdef STRACE
+OPTS = --strace + $(OPTS)
+endif
+
+all: rootfs
+
+appdir:
+	$(APPBUILDER) Dockerfile
+	cp $(PYTEST_INI_FILE) appdir/
+
+rootfs: appdir
+	$(MYST) mkext2 appdir rootfs
+
+rootfs-passed: appdir
+	cat tests.partially_passed | while read line; do rm -f appdir/"$$line"; done
+	$(MYST) mkext2 appdir rootfs-passed
+
+run-passed: rootfs-passed
+	$(MYST_EXEC) rootfs-passed $(OPTS) $(PYTHON3) /app/app.py
+
+run: rootfs
+	(cat tests.partially_passed | xargs -P 8 $(MYST_EXEC) rootfs $(OPTS) \
```

Comment:
Please add comments on why we're doing this here (line 43 - 48) as we don't normally do this in our tests.

Reply:
Added comment